src/test/java/com/epam/wl/dao/BookOrderDAOTest.java

```diff
@@ -41,7 +41,9 @@ void setUp() throws SQLException {
     @Test
     void createOneRow() throws SQLException {
         BookInstance bookInstance = new BookInstance(1, 1);
-        UserOrder userOrder = new UserOrder(1,1, 1, UserOrderStatus.IN_PROGRESS);
+        UserOrder userOrder = new UserOrder(
+                1, 1, "Иван", "Иванов", "ivan@ivan.ru", "Азбука",
+                "Петр Иванов", 1954, UserOrderStatus.IN_PROGRESS);
         BookOption bookOption = BookOption.SUBSCRIPTION;
         bookOrderDAO.create(bookInstance, userOrder, bookOption);
```

Comment:
Why you pass whole entities to DAO method? You need only ids and option

Reply:
Fixed